server/db/user.js

```diff
@@ -50,7 +50,7 @@ const User = db.define('user', {
 });
 
 User.prototype.correctPassword = function (candidatePassword) {
-  return this.Model.encryptPassword(candidatePassword, this.salt) === this.password;
+  return this.constructor.encryptPassword(candidatePassword, this.salt) === this.password;
```

Comment:
asking because I'm not familiar :) but why do we use `constructor` instead of `Model` if this is a class method

Reply:
i thought both should work, but for some reason when I had this.Model it was throwing an error, but was fixed when i made this change.

Reply:
👍